.github/linter.yml

```diff
@@ -0,0 +1,7 @@
+name: Linter
```

Comment:
needs to be under `.github/workflows/linter.yml` to work

Reply:
moved on all